src/apps/index/styles/nav.sass

```
@@ -0,0 +1,87 @@
+@import './variables.sass'
+$nav-width: 450px
+
+body
+  margin-left: $nav-width
+
+  @media (max-width: $mobile-max-width)
+    margin-left: 0
+
+nav
+  background-image: url('https://images.unsplash.com/photo-1502485582557-aee19598572c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1952&q=80')
+  background-size: cover
+  background-position: center
+  height: 100vh
+  width: $nav-width
+  padding: 4rem
+  display: flex
+  flex-direction: column
+  align-items: center
+  justify-content: flex-start
+  font-family: Lobster, cursive, serif
+  position: fixed
+  left: 0
+  top: 0
+
+  @media (max-width: $mobile-max-width)
+    position: relative
+    width: 100%
+    height: 100vh
```

```suggestion
    min-height: 100vh
    height: auto
```
firefox & chrome on my machine do not correctly show the full header,
this plus the other fix in this review solved the problem.